chapters/ch12.md

```diff
@@ -4,7 +4,7 @@
 
 * Choose dev and test sets from a distribution that reflects what data you expect to get in the future and want to do well on. This may not be the same as your training data’s distribution.
 ->
-* Chọn các tập phát triển và tập kiểm tra từ phân phối mà phân phối đó phản ánh dữ liệu nào bạn mong muốn nhận được trong tương lai và muốn dữ liệu đó hoạt động tốt. Phân phối này có thể không giống như phân phối của dữ liệu huấn luyện của bạn.
+* Chọn các tập phát triển và tập kiểm tra sao cho có thể phản ánh dữ liệu bạn gặp phải trong tương lai và muốn hoạt động tốt trên nó. Phân phối này có thể không giống như phân phối của dữ liệu huấn luyện.
```

Comment:
@tiena2cva Bản chỉnh lại của em đã bỏ đi hoàn toàn cái **phân phối** ở câu đầu. Và tự nhiên ở câu thứ 2 lại giải thích **Phân phối này**. Em xem dịch cách tốt hơn ở câu đầu mà vẫn dữ thông tin **tập phápt triển và tập kiểm tra từ một phân phối** nhé.

Reply:
Vâng ạ